modules/virtual_environments/auto-venv/venv_helpers.nu

```diff
@@ -23,6 +23,20 @@ export def venv-is-active [] {
     '__auto_venv' in (overlay list)
 }
 
+export def 'venv-create' [] {
+# Creates a virtual environment under the current directory
```

Comment:
if you want this to be the doc of `venv-create`, i.e. available through `help venv-create`, you might want to put that line just before the `export def` :yum:

Reply:
yeah, that's better :wink: